recipe/run_test.py

```diff
@@ -1,7 +1,7 @@
 import netCDF4
 
 # OPeNDAP.
-url = 'http://omgsrv1.meas.ncsu.edu:8080/thredds/dodsC/fmrc/sabgom/SABGOM_Forecast_Model_Run_Collection_best.ncd'
+url = 'http://geoport-dev.whoi.edu/thredds/dodsC/estofs/atlantic'
```

Comment:
Note to self. Long strings are still segfaulting on some URLs when using `filepath`! See https://github.com/conda-forge/staged-recipes/pull/43